docs/README.TESTING

```diff
@@ -40,11 +40,11 @@ If you use a packaged version of the GD Library (installed from RPM, deb or
 ports, gentoo packages or any other packages format or distribution), please try
 to run the tests using the source releases available at:
 
-http://www.libgd.org/Downloads
+http://github.com/libgd/libgd/release
```

Comment:
this doesn't work either :).  it's "releases", not "release".

go with https://libgd.org/pages/downloads.html for now instead though